CMakeLists.txt

```diff
@@ -63,12 +63,25 @@ FetchContent_Declare(
 
 set(STBIN "off" CACHE STRING "Relic - Build static binaries")
 
-find_package(gmp)
-if (GMP_FOUND)
-  message(STATUS "Found libgmp")
+if(MSVC)
   set(ARITH "gmp" CACHE STRING "")
+  # We'll be using our custom MPIR build instead of GMP for MSVC builds
+  FetchContent_Declare(
+    chia_mpir
+    GIT_REPOSITORY https://github.com/Chia-Network/mpir_gc_x64.git
+    GIT_SHALLOW true
```

Comment:
do we want to pin this to a specific commit?
I would think it would make for fewer moving parts as we can update our MPIR build independently of taking that update in the blspy build

Reply:
We're in control of this repo, we rarely update those DLLs and we would like to take the latest. Furthermore, having inconsistency in what dlls we fetch from this dependency, across repos, would result in issues.